spec/classes/puppet_agent_prepare_spec.rb

```diff
@@ -7,20 +7,10 @@
 
 describe 'puppet_agent::prepare' do
   context 'supported operating system families' do
-    ['Debian', 'RedHat', 'SuSE'].each do |osfamily|
```

Comment:
This test fails for all PE-only platforms. It seems like, considering that and what it's actually testing, it wouldn't hurt to remove SLES here.